sigmf/archive.py

```diff
@@ -21,80 +23,83 @@
 
 
 class SigMFArchive():
-    """Archive a SigMFFile.
+    """Archive one or more `SigMFFile`s.
 
     A `.sigmf` file must include both valid metadata and data.
     If `self.data_file` is not set or the requested output file
     is not writable, raise `SigMFFileError`.
 
     Parameters:
 
-      sigmffile -- A SigMFFile object with valid metadata and data_file
+      sigmffile -- An iterable of SigMFFile objects with valid metadata and data_files
 
       name      -- path to archive file to create. If file exists, overwrite.
```

Comment:
Should this be changed to `path`? It seems like this expects a full path (except for the extension being optional). If that understanding is correct, it would be nice to support multiple input types for this parameter (e.g., `os.PathLike`)

Reply:
Doug Anderson's version also called this path, I will change it to path

Reply:
Changed in 8d25adf